media-sound/musescore/musescore-3.2.3.recipe

```diff
@@ -65,17 +65,17 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	make release PREFIX=$prefix UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
+	make release PREFIX=$appsDir UPDATE_CACHE=FALSE BUILD_PULSEAUDIO=OFF \
```

Comment:
Haven't checked, but couldn't you add BINDIR=$appsDir here? No need to modify the other issues then?